app/stylesheets/alchemy/admin/resource_info.scss

```
@@ -51,13 +65,14 @@
       min-height: var(--form-field-height);
       background: var(--form-field-background-color);
       border-radius: var(--border-radius_medium);
-      @include mixins.truncate(var(--form-right-column-width));
+      @include mixins.truncate(var(--form-right-column-width), $wrap: nowrap);
     }
 
     &.with-icon {
       > p {
         white-space: nowrap;
-        padding-right: 40px;
+        padding-right: var(--spacing-10);
+        max-width: var(--max-width);
```

Would it be an option to set a default value?
```suggestion
        max-width: var(--max-width, 328px);
```